package.json

```diff
@@ -1,35 +0,0 @@
-{
-  "name": "grocy-docker",
-  "version": "3.3.1-1",
-  "description": "ERP beyond your fridge - now containerized",
-  "main": ".",
-  "scripts": {
-    "build": "docker-compose build",
-    "test": "npm run build && npm run test:backend && npm run test:frontend",
```

Comment:
This `package.json` file (and associated `package-lock.json`) is used purely to provide support for Snyk containerfile vulnerability scans.

Although there are some tradeoffs, it seems better overall to perform that as part of GitHub Actions workflows, and so these files are removed in preparation for that.

(see also a related note at https://github.com/grocy/grocy-docker/issues/109#issuecomment-753493188)

These files contained the `grocy-docker` container iteration release number, for no particularly good reason, and so it also simplifies the release process and makes it less error-prone by removing these files.